diff -ur libdvbcsa/src/Makefile.am libdvbcsa-fixed/src/Makefile.am
--- libdvbcsa/src/Makefile.am	2015-06-11 23:42:10.248975300 +0500
+++ libdvbcsa-fixed/src/Makefile.am	2015-06-11 23:47:54.785681600 +0500
@@ -21,5 +21,5 @@
 libdvbcsa_la_SOURCES += dvbcsa_bs_transpose32.c
 endif
 
-libdvbcsa_la_LDFLAGS = -version-info 1:1:0 $(libtool_flags)
+libdvbcsa_la_LDFLAGS = -version-info 1:1:0 -no-undefined $(libtool_flags)
 
